src/formats/formats.js

```diff
@@ -0,0 +1,20 @@
+import { highlighted } from './highlighted';
+
+/**
+ * WordPress dependencies
+ */
+const { registerFormatType } = wp.richText;
+
+function registerNovaBlocksFormats() {
+	[
+		highlighted
```

Comment:
Do we have a list of formats that we want to ship?

Reply:
For the moment we have only the Highlighted style. But I think that in the future we could add at least 1 more this is why I choose to write it like this.